components/LogoNav/LogoNav.tsx

```diff
@@ -11,15 +12,17 @@ import logo from '/public/assets/logo-colored-nav.svg';
  */
 export const LogoNav = (): JSX.Element => {
   return (
-    <Image
-      src={logo}
-      alt="Gridiron Survivor logo"
-      width={1}
-      height={1}
-      priority
-      className="w-20"
-      data-testid="logo-nav"
-    />
+    <Link href="/league/all" passHref>
+      <Image
+        src={logo}
+        alt="Gridiron Survivor logo"
+        width={1}
```

Comment:
width and height of 1px?

Reply:
This is a good question. It seemed the changes were made prior to my change but let me check with the Figma design.

Reply:
Removed width, height, className, and priority and verified design specs adhere to Figma's design.

<img width="845" alt="image" src="https://github.com/user-attachments/assets/ba9b4ea2-cc4b-44e8-b2c4-d9afb20332a6">